src/sentry/sentry_metrics/multiprocess.py

```diff
@@ -332,6 +339,29 @@ def join(self, timeout: Optional[float] = None) -> None:
         self.__producer.close()
 
 
+def valid_metric_name(name: str) -> bool:
+    if not name:
+        return False
+    if len(name) > MAX_NAME_LENGTH:
+        return False
+
+    return True
+
+
+def valid_metric_tags(tags: Mapping[str, str]) -> Tuple[bool, Sequence[str]]:
+    invalid_strs: List[str] = []
+    is_valid = True
+    for key, value in tags.items():
```

Comment:
You could drop to one condition below if you changed this to `for user_str in [*tags.keys(),*tags.values()]:`